synapse/lib/cell.py

```diff
@@ -1854,8 +1854,8 @@ async def setDriveData(self, iden, versinfo, data):
         return self.drive.setItemData(iden, versinfo, data)
 
     @s_nexus.Pusher.onPushAuto('drive:data:del')
-    async def delDriveData(self, iden, vers=None):
-        return self.drive.delItemData(iden, vers=vers)
+    async def delDriveData(self, iden, vers):
+        return self.drive.delItemData(iden, vers)
```

Comment:
Is there still a reason for drive.delItemData / drive._delItemData to use a default vers=None ? It feels like if someone were to pass `vers=None` they would end up with a replay-unsafe nexus event.

Reply:
The implementation of the Drive class is not nexus aware or nexus safe. It's up to the Cell to implement nexus safe APIs.

Reply:
Actually, i like the "whatever is current" use case. I'll divide `delDriveData()` into a top/bottom half where the top half can lookup the current version if it's not specified.

Reply:
good call on splitting it up :)